### PR TITLE
Shared hosting info

### PR DESCRIPTION
**Pull request by terencesimms**
A doc we can link to when running the campaign to close down these old systems
 - how to access your ftp details for shared hosting
 - how to access your shared dbs (whatever info they have in their app, don't wanna give out an IP, and grants are % anyhow)
 - how to back up your mail if still in use.

### CODE REVIEW

**Comment by terencesimms** [comment]
spellcheck seems to be failing on jargon
```
Misspelled words:
<htmlcontent> source/domains/sharedhosting/legacysharedhosting.md: html>body>p
--------------------------------------------------------------------------------
Zippymail
--------------------------------------------------------------------------------

Misspelled words:
<htmlcontent> source/domains/sharedhosting/legacysharedhosting.md: html>body>ul>li
--------------------------------------------------------------------------------
pst
--------------------------------------------------------------------------------
```

**Comment by miff2000** [comment]
Have a read of this guidance I added:

* https://github.com/ukfast/docs.ukfast.co.uk#spell-checking

With ones that are not real words, like `MBOX` or `pst`, I've been surrounding those in back-ticks, so it's evident that they are technically terminology.

With Zippymail, I'd add that to the dictionary file as it's a product of ours. 

With the `CMD` one, is that really an acronym? Assuming it's the windows command prompt, I'd replace it with `Command Prompt`, then it's not assumed that you know what `CMD` means.

Hope that helps?

**Comment by terencesimms** [comment]
it was the mac command key, so slightly different, but backticks are fine
enough for me to get around it. Cheers!

File: source/domains/sharedhosting/files/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+something
```
Comment (miff2000) [inline source/domains/sharedhosting/files/test.txt]: What's this for?

**Review by CountBacula — APPROVED** [review]
LGTM

**Review by miff2000 — COMMENTED** [review]
Could you change the PR to allow changes from maintainers? Just pushing a couple of changes and cleaning up the commit log

File: source/domains/sharedhosting/files/test.txt (hunk 1 above)
Comment (miff2000) [reply source/domains/sharedhosting/files/test.txt]: I'll remove from the commit log

**Pull request merged**